users-service/features/support/env.ls

```diff
@@ -8,7 +8,7 @@ require! {
 db = null
 get-db = (done) ->
   return done db if db
-  MongoClient.connect "mongodb://localhost:27017/space-tweet-users-test", N (mongo-db) ->
+  MongoClient.connect "mongodb://localhost:27017/space-tweet-db-test", N (mongo-db) ->
```

Comment:
Ah, got it. I had assumed we were distinguishing users and tweets on the _collections_ level, but we are doing so on the _database_ level, is that correct?